minotaur.cow

```diff
@@ -0,0 +1,10 @@
+$the_cow = <<"EOC";
+        $thoughts   ^__^
+         $thoughts  ($eyes)
```

Comment:
Odd, must've misread the diff earlier today. Apologies!

(although the earring is kinda cool, haha)